dev.md

```diff
@@ -0,0 +1,45 @@
+## Develop, Release, and Use SQLFlow-in-a-VM
+
+### For Developers
+
+1. Install VirtualBox and Vagrant on a computer with a relatively large memory size.
+1. `git clone https://github.com/sql-machine-learning/desktop && cd desktop`
+1. To build and start the VM: `vagrant up`
```

Comment:
A way to speed up downloading vagrant box using tsinghua mirror 

``` bash
vagrant box add https://mirrors.tuna.tsinghua.edu.cn/ubuntu-cloud-images/bionic/current/bionic-server-cloudimg-amd64-vagrant.box --name ubuntu/bionic64
```
Ref: https://stackoverflow.com/a/22091770, https://segmentfault.com/q/1010000011063709

Reply:
Do you want to add instructions about using the Tsinghua mirror? I agree it is a good idea. How about we merge this and then you create a new PR to add it?

Reply:
> How about we merge this and then you create a new PR to add it?

Sure, I'm not good at Vagrant, so I'm trying to go through as the steps from your PR and leave some comments here. I will add them to a new PR.